src/cursor.rs

```diff
@@ -4,12 +4,13 @@
 //! # #[macro_use] extern crate bson;
 //! # extern crate mongodb;
 //! #
-//! # use mongodb::{Client, ThreadedClient};
+//! # use mongodb::{Connector, ThreadedClient};
```

Comment:
This example was updates so that the test would pass. All other changes to the file are from rustfmt.